app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/TrendingItem.kt

```diff
@@ -50,45 +52,41 @@ fun TrendingItem(
                 .size(Dp60)
                 .padding(end = Dp16)
                 .constrainAs(logo) {
-                    top.linkTo(coinSymbol.top)
-                    bottom.linkTo(coinName.bottom)
+                    top.linkTo(parent.top)
+                    bottom.linkTo(parent.bottom)
```

Comment:
Shall we use `linkTo` to group the `vertical/horizontal` ref, I will apply for next time too 🙏 
```suggestion
                    linkTo(top = parent.top, bottom = parent.bottom)
```